js/wp-seo.js

```diff
@@ -176,7 +176,11 @@
 					frame.on( 'select', function() {
 						var attachment = frame.state().get( 'selection' ).first().toJSON();
 						var img = document.createElement( 'IMG' );
-						img.src = attachment.url;
+						if (attachment.sizes.thumbnail.url) {
```

Comment:
- Needs spacing in parentheses.
- Will these properties be defined if the user chooses something other than an image? Maybe we should make sure the media modal searches for only images?